src/Game.ts

```diff
@@ -3,6 +3,7 @@ import Field from './Field';
 
 class Game {
     board: Board;
+    turn: string = 'white';
```

Comment:
Zróbmy to tak jak board, w tym miejscu jedynie zróbmy przypisanie typu
A w konstruktorze zróbmy:
this.turn = 'white'

Reply:
poprawione